api/src/paths/reference/get/technique-attribute.ts

```diff
@@ -0,0 +1,173 @@
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { getAPIUserDBConnection } from '../../../database/db';
+import { TechniqueAttributeService } from '../../../services/technique-attributes-service';
+import { getLogger } from '../../../utils/logger';
+
+const defaultLog = getLogger('paths/reference');
+
+export const GET: Operation = [getTechniqueAttributes()];
+
+GET.apiDoc = {
+  description: 'Find technique attributes',
+  tags: ['reference'],
+  parameters: [
+    {
+      in: 'query',
+      name: 'methodLookupId',
+      schema: {
+        type: 'array',
+        items: {
+          type: 'string'
+        },
+        minItems: 1
+      },
+      required: true
+    }
+  ],
+  responses: {
+    200: {
+      description: 'Attribute techniques for a method lookup id.',
+      content: {
+        'application/json': {
+          schema: {
+            type: 'array',
+            items: {
+              type: 'object',
+              additionalProperties: false,
+              properties: {
+                method_lookup_id: {
+                  type: 'number'
+                },
+                quantitative_attributes: {
+                  type: 'array',
+                  items: {
+                    type: 'object',
+                    additionalProperties: false,
+                    required: ['method_lookup_attribute_quantitative_id', 'name', 'description', 'min', 'max', 'unit'],
+                    properties: {
+                      method_lookup_attribute_quantitative_id: {
+                        type: 'string',
+                        format: 'uuid'
+                      },
+                      name: {
+                        type: 'string'
+                      },
+                      description: {
+                        type: 'string',
+                        nullable: true
+                      },
+                      min: {
+                        type: 'integer',
+                        nullable: true
```

Comment:
minimum spec

Reply:
negative values might need to be allowed